source/github/github.go

```diff
@@ -40,6 +40,10 @@ type Github struct {
 }
 
 type Config struct {
```

Comment:
Since `Github` struct fields are now unexported, I've used `Config` struct as a way to pass paths and things to the Github driver. `WithInstance()` method sets the values from config.

Reply:
This seems redundant to me. Doesn't the `github.Client` already contain all of this information?

Reply:
I'm not sure I follow? 🤔 I'm using `Config` struct, to set `pathOwner`, `pathRepo` and `path` on the `github` migration source driver. `WithInstance()` takes `Config`, so I thought, this was a good way to do that without need to export the fields. `github.Client` doesn't contain this info.

Reply:
Sorry! I should have noticed this in the last review...

Let's drop the `URL` field (since it's not being used) and rename `PathOwner` and `PathRepo` to `Owner` and `Repo` respectively, since the path prefixes don't make sense anymore...
https://godoc.org/github.com/google/go-github/github#RepositoriesService.GetContents